<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandardView</class>
 <widget class="QWidget" name="StandardView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StandardView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="rebinToolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>23</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select a rebin algorithm for MDEvent data</string>
       </property>
       <property name="text">
        <string>Rebin</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scaleButton">
       <property name="toolTip">
        <string>Invoke plugin to scale the current dataset on any of the given axes.</string>
       </property>
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cutButton">
       <property name="toolTip">
        <string>apply cut filter</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../icons/ViewWidgetsIcons.qrc">
         <normaloff>:/VatesSimpleGuiViewWidgets/icons/pqSlice24.png</normaloff>:/VatesSimpleGuiViewWidgets/icons/pqSlice24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="thresholdButton">
       <property name="toolTip">
        <string>apply threshold filter</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../icons/ViewWidgetsIcons.qrc">
         <normaloff>:/VatesSimpleGuiViewWidgets/icons/pqThreshold24.png</normaloff>:/VatesSimpleGuiViewWidgets/icons/pqThreshold24.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="renderFrame">
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>400</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../icons/ViewWidgetsIcons.qrc"/>
 </resources>
 <connections/>
</ui>
